packages/core/src/vars.css

```diff
@@ -169,6 +169,12 @@
 	--spinner-color-blue: var(--ts-color-blue);
 	--spinner-color-mono: var(--ts-color-slate-lightest);
 	--spinner-color-white: var(--ts-color-white);
+
+	/* Radio button........................................................ */
+
+	--radio-border-selected: 7px;
+	--ts-radio-boxshadow-border-width: inset 0 0 0 var(--radio-border-selected);
+	--ts-radio-unselected-boxshadow-border-width: inset 0 0 0 1.5px;
```

Comment:
why 1.5px?

Reply:
Copied those styles from _ts-ui_. We don't have new designs yet, so it's temporary values. Changed them to standard vars values.

Reply:
Don't let legacy code hold you back ;)